Models/Pokemon.swift

```diff
@@ -10,31 +10,31 @@ import SwiftUI
 import CoreLocation
 
 struct Pokemon: Hashable, Codable, Identifiable {
-    var id: String
-    var number: Int
+    var id: Int
+    var number: Int?
```

Comment:
be careful, if some props are optional in the JSON data, the property in the struct should be optional, or it will crash when parsing

Reply:
In this case, number isn't optional. Nevermind the json, that is outdated as hell. The data in `pokemonData.swift` is the real thing

Reply:
Okay no problem, you can change that back again, when the JSON is with all the full data